projects/js-packages/publicize-components/src/components/social-previews/instagram.js

```diff
@@ -34,7 +34,10 @@ export function Instagram( props ) {
 			<Notice
 				hideCloseButton
 				actions={ [
-					<ExternalLink href={ getRedirectUrl( 'jetpack-social-share-to-instagram' ) }>
+					<ExternalLink
+						key="learn-more"
```

Comment:
Why do we need this one?

Reply:
The eslint rule sees "React element inside array" and triggers, it doesn't know that this array will never be modified or sorted so the key will never really matter.

As I mentioned in the PR summary, most of the key additions in the PR are like this.

Reply:
Never mind, I missed that it was an array.